### PR TITLE
use local model if no model server endpoint given

### PR DESCRIPTION
**Pull request by jichenjc**
related to #375 
Signed-off-by: jichenjc <jichenjc@cn.ibm.com>

### CODE REVIEW

File: cmd/exporter.go [hunk 1]
```diff
@@ -44,9 +44,10 @@ const (
 )
 
 var (
-	address                      = flag.String("address", "0.0.0.0:8888", "bind address")
-	metricsPath                  = flag.String("metrics-path", "/metrics", "metrics path")
-	enableGPU                    = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
+	address     = flag.String("address", "0.0.0.0:8888", "bind address")
+	metricsPath = flag.String("metrics-path", "/metrics", "metrics path")
+	enableGPU   = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
+	// by default it should be "http://kepler-model-server.monitoring.cluster.local:8100/model" if we use model server
```
Comment (rootfs) [inline cmd/exporter.go:50]: No need for the comment, just set the default value for modelServerEndpoint

Comment (rootfs) [inline cmd/exporter.go:50]: the comment line changes the indent. Can you move it to the end of the next line?

File: pkg/collector/metric/utils.go [hunk 2]
```diff
@@ -78,9 +77,7 @@ func getEstimatorMetrics() []string {
 	ratio.InitMetricIndexes(names)
 
 	// do not try to inilialize estimator if it was not configured
-	if config.ModelServerEndpoint != "" {
-		model.InitEstimateFunctions(names, NodeMetadataNames, NodeMetadataValues)
-	}
+	model.InitEstimateFunctions(names, NodeMetadataNames, NodeMetadataValues)
```
Comment (rootfs) [inline pkg/collector/metric/utils.go:80]: this forces to use estimator, are we ready for that @sunya-ch ?

Comment (jichenjc) [reply pkg/collector/metric/utils.go:80]: check below , there is ` r.Endpoint` check will do same thing
but this change won't block sidecar usage if endpoint is empty

Comment (rootfs) [reply pkg/collector/metric/utils.go:80]: per Pang, this is ok

**Comment by rootfs** [comment]
@jichenjc can you rebase? It is ready to merge

**Comment by jichenjc** [comment]
@marceloamaral @sunya-ch please let me know your comments, thanks

**Review by rootfs — APPROVED** [review]

**Pull request merged**